Tetris/Tetris.h

```diff
@@ -38,6 +34,8 @@ class Tetris {
     void key_right(){ if (isActive && !isPaused) shift(MoveTetromino::Right); }
     void key_space(){ if (isActive && !isPaused) letFall = true; }
     void key_g()    { if (isActive && !isPaused) (ghosting = !ghosting) ? updateGhost() : eraseGhost(); }
+    void key_q()    { quit = true; startAnotherGame = isActive = isPaused = false; }    // quit game
```

Comment:
by now two or more flags needs to be checked or set to identify / set a state --> states shall be modeled in enum class

Reply:
introduced proper state model and implemented state machine